system_upgrade.py

```diff
@@ -125,6 +126,20 @@ def checkDNFVer():
     if DNFVERSION < StrictVersion("1.1.0"):
         raise CliError(_("This plugin requires DNF 1.1.0 or later."))
 
+def enable_blanking(arg=10):
+    try:
+        tty = open('/dev/tty0', 'wb')
+    except Exception:
+        return
+    tty.write(b'\33[9;' + str(arg).encode('ascii') + b']')
+
+def maybe_disable_blanking():
+      term = os.getenv('TERM', '')
+      if not (term.startswith('linux') or term.startswith('con')):
+             return
+      atexit.register(enable_blanking)
```

Comment:
This line seemed harmless, but I wonder why we're doing it at all? We want to disable blanking for the whole duration of the upgrade, and then reboot immediately, so why are we trying reset things back at the end? Less code means fewer bugs, I think we don't need to do this.